recOrder/plugin/workers/acquisition_workers.py

```diff
@@ -872,9 +873,16 @@ def work(self):
                                              zend=self.calib_window.z_end,
                                              zstep=self.calib_window.z_step,
                                              save_dir=self.snap_dir,
-                                             prefix=self.prefix)
+                                             prefix=self.prefix,
+                                             keep_shutter_open_channels=True,
+                                             keep_shutter_open_slices=True)
 
             self._check_abort()
+
+            # set acquisition order to channel-first
+            settings['slicesFirst'] = False
```

Comment:
Sorry that my [#162](https://github.com/mehta-lab/recOrder/pull/162) code caused this conflict, but you will need `self.settings` here.

Reply:
Not your fault! Great catch.